src/applications/pensions/config/chapters/06-additional-information/index.js

```diff
@@ -1,3 +1,18 @@
-export { directDeposit } from './directDeposit';
-export { usingDirectDeposit, accountInformation } from './accountInformation';
-export { otherPaymentOptions } from './otherPaymentOptions';
+import { directDeposit } from './directDeposit';
+import { accountInformation } from './accountInformation';
+import { otherPaymentOptions } from './otherPaymentOptions';
+import { supportingDocuments } from './supportingDocuments';
+import { documentUpload } from './documentUpload';
+import { fasterClaimProcessing } from './fasterClaimProcessing';
+
+export const additionalInformation = {
```

Comment:
I do like this export here.  It cleans up the import back on form.js to be only:
import { additionalInformation } from './chapters/06-additional-information';